### DISCUSSION TITLE
Is it possible to prefix the generated operation's graphql string?

### DISCUSSION THREAD

**Discussion by gmencz** [dotansimha/graphql-code-generator#4094] [General] [answered]
So, say we have the following operation:
```js
const USER = gql`
  query User {
    user(username: "@ada") {
      name
    }
  }
`;
```

Is there a way to tell the generator to prefix the query with some string e.g "Local"? The generated document would be the following:
```js
const UserDocument = gql`
  query Local_User {
    user(username: "@ada") {
      name
    }
  }
`;
```

**Comment by dotansimha** [comment, marked as answer]
At the moment, no. GraphQL Code Generator doesn't deal with transforming your GraphQL schema or your GraphQL operations. It just generates code based on it.
